project/Settings.scala

```diff
@@ -38,11 +38,11 @@ object Settings {
     val scalaJQuery  = "1.0"
 
     // Java libraries
-    val scalaZ       = "7.2.2"
+    val scalaZ       = "7.2.10"
     val scalaZStream = "0.8.6a"
```

Comment:
Probably never, `http4s` is going to be exclusively `cats/fs2` starting in 0.17.x